content/en/examples/policy/restricted-psp.yaml

```diff
@@ -22,8 +22,9 @@ spec:
     - 'projected'
```

Comment:
this also sets `requiredDropCapabilities:["ALL"]` which doesn't match where we ended up for restricted, right?

Reply:
both of these are pre-existing, fine if you want to address in a follow-up

Reply:
Let's defer that one to https://github.com/kubernetes/website/pull/27882